### DISCUSSION TITLE
Daily standup (Friday, March 27)

### DISCUSSION THREAD

**Discussion by paotg123** [antonieto/ibm-dashboard#77] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing jiji

**What are your priorities for the next 24 hours?**:
 - Start working on interface mockup

**Blockers**:
 - None

**Shoutouts**:
 - @Int-Angel I miss him
